README.md

```diff
@@ -30,30 +30,8 @@ Guitarix can also load up LAPSPA and LV2 plugins to comply the effect chain.
 - git clone https://github.com/brummer10/guitarix.git
 - cd guitarix
 - git submodule update --init --recursive
+- xargs -a dependencies.txt sudo apt-get install -y
```

Comment:
Unfortunately this only works on debian based distributions, so it couldn't become the general build rule.